src/demos/calcite-select.html

```diff
@@ -6,6 +6,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Calcite Select</title>
     <style>
+     .parent {
```

Comment:
Looks like we're repeating this pattern across multiple demo pages.
We should add this to demo.css.
Also it might be good to make a use class names that are semantically more grid oriented...or use consider using table layout.

Reply:
Will be best to get through all the demo pages component variants, then go through and update them all at once.

Reply:
@ManahariDahal I can help with centralizing the styles if you want a hand. Should be a pretty straightforward find and replace in most cases.

Reply:
Ah...I see. Looks like there are a lot of one-off styles per demo page.